diff/cert.go

```diff
@@ -85,6 +85,18 @@ func (sc *Syncer) createUpdateCertificate(
 
 	// found, check if update needed
 	if !currentCertificate.EqualWithOpts(certificateCopy, false, true) {
+		currentSNIs, err := sc.currentState.SNIs.GetAllByCertID(*currentCertificate.ID)
+		if err != nil {
+			return nil, errors.Wrapf(err, "error looking up current certificate SNIs %v",
+				certificate.Identifier())
```

Comment:
Personally since wrapping became available in `fmt` in Go `v1.14` I would take this opportunity to use the standard library wrapping instead, to stop the growth of the 3rd party library which seems to be outmoded at this point:

```suggestion
			return nil, fmt.Errorf("error looking up current certificate SNIs %v: %w", certificate.Identifier(), err)
```

I don't consider this a blocker, but just for your considerations :thinking: